e2e-tests/e2eTest.spec.ts

```diff
@@ -65,65 +67,29 @@ void describe('e2e-tests', async () => {
 			outputs.cacheTableName,
 		)({ iccid: iccidNotExisting, simExisting: false })
 		//put noHistory SIM in DB
-		await seedingDBFunction({
+		await seedDb({
 			iccid: iccidNoHistory,
 			usageTimestamp: now,
 			simDetails: { usedBytes: 0, totalBytes: 1000 },
 		})
 	})
-	void it(`should return statusCode 200, cache max-age=300 and correct body for iccid: ${iccidNewWL}`, async () => {
-		await seedingDBFunction({
-			iccid: iccidNewWL,
-			usageTimestamp: now,
-			simDetails: { usedBytes: 0, totalBytes: 1000 },
-		})
-		await seedTimestreamFunc({
-			usage: usageLastHour,
-			iccid: iccidNewWL,
-			dbName,
-			tableName,
-		})
-		/*
-			Timestream is seeded with the following values for the last hour: [1, 3, 67, 1, 2, 3, 5, 7, 2, 1, 42, 4],
-			and dataUsagePerTimespan is calculated by summing the values in the array. They should all be 138 since 
-			we only have data for the last hour.
-		*/
-		const req = await fetchDataFunc(iccidNewWL)
-		const expectedCacheControl = 'public, max-age=300'
-		const responseBody = await req.json()
-		assert.equal(req.headers.get('cache-control'), expectedCacheControl)
-		assert.equal(req.headers.get('Access-Control-Allow-Origin'), '*')
-		assert.equal(req.status, 200)
-		assert.deepEqual(responseBody, {
-			dataUsagePerTimespan: {
-				lastDay: 138,
-				lastHour: 138,
-				lastMonth: 138,
-				lastWeek: 138,
-			},
-			ts: now.toISOString(),
-			usedBytes: 0,
```

Comment:
@Lenakh97 why is this `0`? Shouldn't that also be `138`?

Reply:
In https://github.com/bifravst/sim-details/pull/193/commits/639d7f53040b2abd7c13f8ae6939659a5c52c273 I've further separate this and moved the seeders to a separate file.